unit/debug-t.cc

```diff
@@ -0,0 +1,132 @@
+#include <gtest/gtest.h>
+#include <cstring>
+
+#include "kickcat/Link.h"
+#include "kickcat/DebugHelpers.h"
+#include "Mocks.h"
+
+using ::testing::Return;
+using ::testing::_;
+using ::testing::Invoke;
+using ::testing::InSequence;
+
+using namespace kickcat;
+
+struct SDOAnswer
+{
+    mailbox::Header header;
+    mailbox::ServiceData sdo;
+    uint8_t payload[4];
+} __attribute__((__packed__));
+
+class FullLink : public testing::Test
+{
+public:
+
+
+
+    template<typename T>
+    void handleReply(std::vector<T> answers, uint16_t replied_wkc = 1)
```

Comment:
The `FullLink` class seems to be an extract of the `BusTest` class. To avoid code duplication it may be better to move the tests `send_get_register` and `send_write_register` into bus-t

What do you think @Siviuze ?

Reply:
Yes, that or create another file that can be use by both bus-t and debug-t. Note: a better approach seems to be to mock the Link class to avoid managing the whole communication. It may be worth to wait for your Link refactoring.